packages/hint-manifest-icons/package.json

```diff
@@ -14,7 +14,8 @@
     "@hint/utils-i18n": "^1.0.3",
     "@hint/utils-types": "^1.0.1",
     "image-size": "^0.8.3",
-    "image-type": "^4.1.0"
+    "image-type": "^4.1.0",
+    "lodash": "^4.17.15"
```

Comment:
IMO we should avoid adding dependencies when possible. I think we can get away without lodash for this change.